Lib/test/test_socket.py

```diff
@@ -1158,9 +1160,9 @@ def testIPv6toString(self):
         assertInvalid('1:2:3:4:5:6')
         assertInvalid('1:2:3:4:5:6:7:8:0')
         # bpo-29972: inet_pton() doesn't fail on AIX
-        if not sys.platform.startswith('aix'):
+        if AIX:
```

Comment:
Must be `ìf not AIX`?

Reply:
you are right, blind in one eye. But AIX does not "Invalid" assertInvalid('1:2:3:4:5:6:'), so I'll move that, and change the other test back, behind "not AIX".